happy-travel/edo
# Markup by country of location of the hotel for a specific supplier

Issue: https://github.com/happy-travel/agent-app-project/issues/1376
Insomnia: Edo -> Admin -> Markups -> Supplier Markups
Tested through unit tests

Test case:

1) Add supplier destination country markup through POST endpoint ../admin/supplier-markups and check validators
2) Start availability search for hotel from country for specific supplier which markup was added before
3) Check if markup was applied
JSON example:
{
"description": "test",
"value": 5,
"destinationScopeId": "RU",
"destinationScopeType": 2,
"supplierCode": "jumeirah"
}
